.gitignore

```diff
@@ -42,3 +42,4 @@ master/setuptools_trial*
 master/buildbot/www/static/js/default/
 sandbox
 .idea/
+/.venv/
```

Comment:
absolute paths in .gitignore files are considered relative to the location of the .gitignore file itself.

See https://git-scm.com/docs/gitignore